code/game/gamemodes/clock_cult/clock_structures.dm

```diff
@@ -341,16 +341,19 @@
 		if(!(target in validtargets))
 			lose_target()
 		else
-			target.adjustFireLoss(!iscultist(target) ? damage_per_tick : damage_per_tick * 2) //Nar-Sian cultists take additional damage
-			if(ratvar_awakens && target)
-				target.adjust_fire_stacks(damage_per_tick)
-				target.IgniteMob()
-			setDir(get_dir(get_turf(src), get_turf(target)))
+			if(!target.null_rod_check())
```

Comment:
To go with the below it should still damage if ratvar is alive.
Also, it'd be nice if it'd switch target off the chaplain/whoever's holding the holy weapon instead of just trying fruitlessly to kill them forever.